lib/plugins/aws/package/compile/functions/index.js

```diff
@@ -279,6 +279,19 @@ class AwsCompileFunctions {
       delete newFunction.Properties.VpcConfig;
     }
 
+    if (functionObject.reservedConcurrency) {
+      if (_.isInteger(functionObject.reservedConcurrency)) {
+        newFunction.Properties.ReservedConcurrentExecutions = functionObject.reservedConcurrency;
+      } else {
+        const warningMessage = [
+          'WARNING! You should use integer as reservedConcurrency value on function: ',
+          `${newFunction.Properties.FunctionName}`,
+        ].join('');
+
+        this.serverless.cli.log(warningMessage);
```

Comment:
Why waring?
it would be better to show error and stop the Serverless process if reservedConcurrency can handle only integer value.

Reply:
You have right! Error will be more visible and informative. I changed this.